server/src/main/java/org/elasticsearch/search/suggest/phrase/DirectCandidateGeneratorBuilder.java

```diff
@@ -490,7 +490,7 @@ public String toString() {
             toXContent(builder, EMPTY_PARAMS);
             return Strings.toString(builder);
         } catch (Exception e) {
-            return "{ \"error\" : \"" + ExceptionsHelper.detailedMessage(e) + "\"}";
+            throw new RuntimeException(e);
```

Comment:
Do we have a test for this failure case so we know the response is not already too far written to throw/serialize an exception like this?

Reply:
We do have integration tests in `SearchSuggestIT` with failure cases, but they don't look related to a failure to serialize content. I'd be happy to add a new test - any ideas how to provoke a failure when serialising a `DirectCandidateGeneratorBuilder` to XContent?